modules/wildfly/pom.xml

```diff
@@ -95,13 +95,12 @@
             <artifactId>hibernate-envers</artifactId>
             <optional>true</optional>
         </dependency>
-        <!-- This is to satisfy Neo4J's custom Lucene,
-          not to be confused with the one used by Hibernate Search -->
+        <!-- This is to satisfy Neo4J's custom Lucene: might not match the needs of Hibernate Search -->
```

Comment:
not really. In this section we don't pick the version, we're just adding the Lucene dependencies so that they get copied into the Neo4J modules.